### ISSUE TITLE
V8 - Not possible to add a doctype property only using keyboard

### ISSUE THREAD

**Issue by clausjensen** [umbraco/Umbraco-CMS#5524]
When adding a property to a document type, you can navigate through the dialog just fine using the keyboard (tabbing).
After adding a name for the property and then selecting the property editor (Textstring for example) - the dialog has lost its focus which means you cannot move on to the next field using the keyboard. This means you will not be able to add/save the property without using the mouse.

Reproduction
------------

- Create a document type
- Add a tab
- Add a property
- Using keyboard, give it a name and navigate to the next field.
- Navigate to the property editor selection and find "Textstring"
- Hit 'Enter' to select it
- Now you are not able to continue using only the keyboard.

**Comment by kjac** [comment]
Oh my! Didn't know this issue existed... it's fixed in #5703

**Comment by clausjensen** [comment]
@kjac I was wondering when you'd pick this up 😃 
Thanks for fixing!

**Comment by kjac** [comment]
😋
